app/models/registration/requests/IndividualWithoutId.scala

```diff
@@ -34,6 +36,17 @@ object IndividualWithoutId {
   implicit lazy val writes: OWrites[IndividualWithoutId] = new OWrites[IndividualWithoutId] {
 
     def writes(o: IndividualWithoutId): JsObject =
+      val email = o.contact.email
+      val details = o.contact.phone match {
+        case Some(phone) => Json.obj(
+          "emailAddress" -> email,
+          "phoneNumber" -> phone
+        )
+        case None => Json.obj(
+          "emailAddress" -> email
+        )
+      }
```

Comment:
Can you please change Contact to ContactDetails and use the fields as in the documentation, i.e. instead of email and phone you can use emailAddress and phoneNumber. This way you will not need to override the fields and the code is simpler.